commands/flags.go

```diff
@@ -116,6 +116,12 @@ var DaemonFlags = append(DealFlags, append(CommonFlags, append(EndpointFlags, []
 		EnvVars: []string{"DEALBOT_WORKERS"},
 		Value:   1,
 	}),
+	altsrc.NewIntFlag(&cli.IntFlag{
+		Name:    "tags",
+		Usage:   "comma separated tag strings",
+		EnvVars: []string{"DEALBOT_TAGS"},
+		Value:   1,
```

Comment:
should this be a string array flag?

Reply:
Probably, but I did not know how environ vars work with that.  Will an environ var like "tag1, tag2" be converted to ["tag1", "tag2"] ?

Reply:
i think you want https://github.com/urfave/cli/blob/master/flag_string_slice.go#L72 "StringSliceFlag"
and yes, it should auto do the splitting of commas then

Reply:
Yes, just found out that was there.  Thank you.